sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -734,7 +734,7 @@ private[sql] object QueryExecutionErrors extends QueryErrorsBase {
   def failedToFindDataSourceError(
```

Comment:
Let's make it consistent w/ the error class name:
```suggestion
  def dataSourceNotFoundError(
```

Reply:
Thanks!